src/main/java/com/softserve/actent/model/dto/CreateUserDto.java

```diff
@@ -0,0 +1,18 @@
+package com.softserve.actent.model.dto;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+public class CreateUserDto {
+
+    @NonNull
+    private String firstName;
+    @NonNull
+    private String lastName;
```

Comment:
Check out (read about) for @NotEmpty @NotBlank annotations, decide which needed

Reply:
These @NonNull are Lombok's annotations to create constructor for annotated fields; we'll also check out your suggestion